roles/edpm_telemetry/defaults/main.yml

```diff
@@ -42,3 +42,7 @@ edpm_telemetry_image_download_retries: 5
 edpm_telemetry_old_tripleo_compute_sevices:
   - tripleo_ceilometer_agent_compute.service
   - tripleo_ceilometer_agent_ipmi.service
+# Release of Kepler to be installed (Browse all releases here https://github.com/sustainable-computing-io/kepler/releases)
+edpm_telemetry_kepler_release: "0.7.11"
+# URL that provides Kepler rpm to install on compute nodes
+edpm_telemetry_kepler_image: "quay.io/sustainable_computing_io/kepler:release-{{ edpm_telemetry_kepler_release }}"
```

Comment:
we can't just run containers from quay.io in every deployment of our product. This would need to be managed how every other container image is managed in edpm-ansible. Unless this discussion has already happened somewhere, we need to hold off on this PR until this is resolved.

Reply:
I see nova-compute image is being pulled from quay here https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/roles/edpm_nova/defaults/main.yml#L33 

Node exporter follows the same approach for deployment https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/roles/edpm_telemetry/defaults/main.yml#L26 and it pulls image from the `prometheus` repository.

Seems almost all containers are being pulled from quay.

Reply:
If it's meant to be handled like the other images, then there needs to be a related PR for openstack-operator that adds support the image with a $RELATED_IMAGE* var, and handling in OpenStackVersion. You should be able to review what's there for the node exporter image and copy that same pattern.

Reply:
@slagle https://github.com/openstack-k8s-operators/openstack-operator/pull/1034